demo/src/routes/menu/Versions.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+	import {get, writable} from '@amadeus-it-group/tansu';
+	import type {ReadableSignal, Writable} from '@amadeus-it-group/tansu';
+	import type {Directive} from '@agnos-ui/svelte/types';
+	import {createHasFocus} from '@agnos-ui/svelte/services/focustrack';
+	import {onMount} from 'svelte';
+	import {page} from '$app/stores';
+	import {pathToRoot$, selectedFramework$} from '$lib/stores';
+
+	interface Version {
+		folder: string;
+		version: string;
+	}
+
+	const versionsParsed: Version[] = [{folder: 'latest', version: 'Latest'}];
+
+	onMount(() => {
+		fetch('https://amadeusitgroup.github.io/AgnosUI/versions.json').then((res) => {
+			res.json().then((result) => {
+				versionsParsed.push(...result);
+			});
+		});
+		({hasFocus$, directive} = createHasFocus());
+	});
+
+	const currentVersion$: Writable<Version> = writable(versionsParsed[0]);
```

Comment:
currentVersion initial value should probably be built using AGNOSUI_VERSION, something like this:
```typescript
  const currentVersion$ = writable(versions.find((version) => version.version === `v${import.meta.env.AGNOSUI_VERSION}`) ?? versions[0]);
```